connectors/v2/pitchfork.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+/* global Connector */
+
+Connector.artistSelector = '.track-title .artist';
+
+Connector.trackSelector = '.track-title .title';
+
+Connector.isPlaying = function () {
+	console.log($('.playback-button > div').attr('title'));
+	return $('.playback-button > div').attr('title') == 'Pause Track';
```

Comment:
@inverse do you think it's worth adding an jshint option to `.jshintrc` to check strict equality? http://jshint.com/docs/options/#eqeqeq